.github/workflows/publish-release.yml

```diff
@@ -56,4 +56,4 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.BALLERINA_BOT_TOKEN }}
         run: |
           curl -fsSL https://github.com/github/hub/raw/master/script/get | bash -s 2.14.1
-          bin/hub pull-request -m "[Automated] Sync master after "$VERSION" release
+          bin/hub pull-request -m "[Automated] Sync master after "$VERSION" release"
```

Comment:
Shouldn't we use single quotes inside the double quotes of vice-versa here? I'm not sure whether this works or not.

Reply:
Fixed in here https://github.com/ballerina-platform/module-ballerina-io/pull/19/commits/d800e29dbd058a690b50483c0b6dddd1f500ed81